code/game/turfs/turf.dm

```diff
@@ -50,21 +52,39 @@
 				return TRUE
 
 	return FALSE
-
-/turf/New()
-	..()
-	for(var/atom/movable/AM as mob|obj in src)
-		spawn( 0 )
-			src.Entered(AM)
-			return
-
+/turf/Initialize(mapload, ...)
+	. = ..()
 	if(dynamic_lighting)
 		luminosity = 0
 	else
 		luminosity = 1
 
+	if (mapload && permit_ao)
+		queue_ao()
+
+	if (z_flags & ZM_MIMIC_BELOW)
```

Comment:
add this flag to space tiles, i want them to have zmimic functionality too

Reply:
Space tiles will have undefined behaviour if using zmimic. Correct way to handle it would be to make a new unsimulated turf for vacuum. Space would only be bottom of the map. 

ChangeTurf doesnt support space above things so it's also good for long term correctness

Reply:
That's fine to do if you can do it procedurally. Making this work in space is the most important part of this project

it can be done in a second PR though